### DISCUSSION TITLE
Discord link broken

### DISCUSSION THREAD

**Discussion by gklijs** [databendlabs/openraft#739] [General]
Hi, looks like an interesting project. I already started a project that uses it, no questions yet. But I tried the Discord link from the readme and it's not working for me.

**Comment by drmingdrmer** [comment]
Thanks for informing me.

The corrected discord link is:
https://discord.gg/yqSsrA5M

I'm new to Discord and wasn't aware that an invitation link should be used instead of the channel link.

I hope everything goes smoothly with your project. :D

**Reply by siennathesane** [reply]
The link isn't working again, can you please create a permanent link?

**Reply by drmingdrmer** [reply]
Oh...my. I still do not get how discord works...Let me fix it. Thank you!

**Reply by drmingdrmer** [reply]
https://discord.gg/ZKw3WG7FQ9
This is a invitation link without expire. Hope this time I get it correctly. :|
